dynamo/__init__.py

```diff
@@ -24,12 +24,50 @@
 # *
 # **************************************************************************
 
-import pyworkflow.em
+from os.path import join
+import pyworkflow.em as pwem
+import pyworkflow.utils as pwutils
+from .constants import *
 
 _logo = "icon.png"
 _references = ['CASTANODIEZ2012139']
 
-class Plugin(pyworkflow.em.Plugin):
-    pass
+class Plugin(pwem.Plugin):
+    _homeVar = DYNAMO_HOME
+    _pathVars = [DYNAMO_HOME]
+    # _supportedVersions =
 
-pyworkflow.em.Domain.registerPlugin(__name__)
+    @classmethod
+    def _defineVariables(cls):
+        cls._defineEmVar(DYNAMO_HOME, 'dynamo-{}'.format(DYNAMO_VERSION))
+
+    @classmethod
+    def getEnviron(cls):
+        """ Create the needed environment for Dynamo programs. """
+        environ = pwutils.Environ(os.environ)
+        dyn_home = cls.getHome()
+        environ.update({
+            'PATH': dyn_home + '/matlab/bin:' + dyn_home + '/matlab/src:' + dyn_home + '/cuda/bin:' + dyn_home +
+                    '/mpi:${PATH}',
+            'LD_LIBRARY_PATH': dyn_home + '/MCRLinux/runtime/glnxa64:' + dyn_home + '/MCRLinux/bin/glnxa64:' +
+                               dyn_home + '/MCRLinux/sys/os/glnxa64:${LD_LIBRARY_PATH}',
```

Comment:
Just a question, do we actually need this at the end? ${LD_LIBRARY_PATH}

Reply:
As you said, it ins't necessary with the BEGIN functionality.